src/module/token/royalty/RoyaltyERC1155.sol

```diff
@@ -1,32 +1,304 @@
 // SPDX-License-Identifier: Apache-2.0
 pragma solidity ^0.8.20;
 
+import {ModularModule} from "../../../ModularModule.sol";
+
 import {Role} from "../../../Role.sol";
-import {RoyaltyERC721} from "./RoyaltyERC721.sol";
 
-contract RoyaltyERC1155 is RoyaltyERC721 {
+import {BeforeBatchTransferCallbackERC1155} from "../../../callback/BeforeBatchTransferCallbackERC1155.sol";
+import {BeforeTransferCallbackERC1155} from "../../../callback/BeforeTransferCallbackERC1155.sol";
+import {IInstallationCallback} from "../../../interface/IInstallationCallback.sol";
+
+import {ICreatorToken} from "@limitbreak/creator-token-standards/interfaces/ICreatorToken.sol";
+
+import {ITransferValidator} from "@limitbreak/creator-token-standards/interfaces/ITransferValidator.sol";
+import {ITransferValidatorSetTokenType} from
+    "@limitbreak/creator-token-standards/interfaces/ITransferValidatorSetTokenType.sol";
+import {TOKEN_TYPE_ERC1155} from "@limitbreak/permit-c/Constants.sol";
+
+library RoyaltyStorage {
+
+    /// @custom:storage-location erc7201:token.royalty
+    bytes32 public constant ROYALTY_STORAGE_POSITION =
+        keccak256(abi.encode(uint256(keccak256("token.royalty")) - 1)) & ~bytes32(uint256(0xff));
```

Comment:
include the token spec in case of collision, since the implementation is token specific: `token.royalty.erc1155`